daos/rinkeby/nectardao.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Nectar DAO",
+  "Avatar": "0xd04FaA2AFdb34575F10B116361d19A394965f858",
+  "DAOToken": "0x932170dFe646D908371E099DB51FdfF582f2de75",
+  "Reputation": "0x91adb655CB783d9Ff6324E1f8b105b8FA8C765a1",
+  "Controller": "0x0b7583C19FaeF378eD254Fb5111DaBE77022bdDF",
+  "Schemes": [
+    {
+      "name": "GenericScheme",
+      "alias": "GenericSchemeENS",
+      "address": "0x84dc738B6bc14952d4aaa098E4085D913777F4D0"
+    },
+    {
+      "name": "GenericScheme",
+      "alias": "GenericSchemeTrading",
+      "address": "0x8f3D6363b891e6286490336AefcC0Fd5E9091DD2"
+    },
+    {
+      "name": "GenericScheme",
+      "alias": "GenericSchemeWhitelist",
+      "address": "0x4169DbB4E024d605423B943f7373Abd2E0D3D006"
+    },
+    {
+      "name": "ContinuousLocking4Reputation",
+      "alias": "InfiniteReputation4Token",
+      "address": "0x9a3dfb05d46357D88273EC14B664214a2237dC5b"
+    },
+    {
+      "name": "Auction4Reputation",
+      "alias": "GENAuction",
+      "address": "0x1f75634317f108dAeF3Db3B6bedc767667464583"
+    },
+    {
+      "name": "ReputationFromToken",
+      "alias": "ReputationFromToken",
+      "address": "0x8401576723F4c4840f465b30e1F0a4A51508874c"
+    }
+  ],
+  "arcVersion": "0.0.1-rc.28"
```

Comment:
do we have universal scheme mapping for 28 ?

Reply:
No but the addresses of the Universal schemes are the same as v24 so it shouldn't matter.